easybuild/easyconfigs/p/ParallelIO/ParallelIO-2.5.3-iompi-2020a.eb

```diff
@@ -1,14 +1,14 @@
 easyblock = 'ConfigureMake'
 
-name = 'ParallelIO'
+name = 'pio'
```

Comment:
The name is `ParallelIO` but set the sources to : sources = ['pio-%(version)s.tar.gz']